src/plugins/index_pattern_editor/public/components/index_pattern_editor_flyout_content.tsx

```diff
@@ -225,52 +238,28 @@ const IndexPatternEditorFlyoutContentComponent = ({
       let newRollupIndexName: string | undefined;
 
       const fetchIndices = async (query: string = '') => {
+        const currentRequestRef = {};
+        currentLoadingMatchedIndicesRef.current = currentRequestRef;
```

Comment:
I added this to avoid a race-condition where we can override more recent request with older response